test/snapshot/__snapshots__/graceful.test.js.snap

```diff
@@ -837,18 +838,19 @@ Program {
           "kind": "propertystatement",
           "properties": Array [
             Property {
-              "attrGroups": Array [],
+              "attrGroups": null,
```

Comment:
This diff and the one right below looks a bit off, doesn't it?

Reply:
The test in question is

```javascript
 it("test class", function () {
      expect(test.parseEval("class foo { foo const A = 1 ")).toMatchSnapshot();
    });
```

My understanding of this test suite is to make sure that the parser does not throw an error when providing invalid PHP and when setting the option `suppressErrors` to true. 

If that understanding is correct, I am not sure I need to try to "fix" how an invalid PHP class gets parsed.

Reply:
Ah, I didn't check what the test was actually doing. Sounds reasonable!